src/power/config.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable */
+
+import { Data } from '../vendor/Data';
+
+const TESTS = [
+  {
+    id: 'cpu',
+    label: 'CPU',
+    filter: { eq: { test: 'cpu' } },
```

Comment:
It seems like for something like an id (presumably used for internal purposes) you can just use the label?

Reply:
In general, I separate the machine-friendly `id` from the humane `label` because they too-often diverge. 

Humans like spaces and capitalization, and sometimes like names to change, while machines like consistency (eg all lowercase, or not changing) and standardization (eg use third party names, like from perfherder).  For example; I have no idea what `proportional` means: I will learn shortly, and it will be easy to change.